browse-scaler/src/cmr.js

```diff
@@ -38,18 +52,27 @@ exports.getBrowseImageFromConcept = async concept => {
   }
 };
 
+/**
+ * getGranuleLevelBrowseImage: Given a or collection id, get the first associated granule
+ * @param {String} granuleId CMR concept-id. This can be a collection _or_ granule id
+ * @returns {String} any image links found. If a collection id is supplied, this will
```

Comment:
Is this a `String` or and array of `String`s (`String[]`)?

Reply:
just one - I updated the comment here